test/spec/spectest.cpp

```diff
@@ -597,10 +613,10 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
         return;
       }
       case CommandID::AssertReturn: {
-        const auto &Action = Cmd["action"s];
-        const auto &Expected = Cmd["expected"s];
-        const auto ActType = Action["type"].Get<std::string>();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
+        const uint64_t LineNumber = Cmd["line"];
+        const simdjson::dom::object Action = Cmd["action"].get_object();
```

Comment:
Use reference instead of copying a new data structure.

Reply:
Same as above